src/test/java/core/basesyntax/RegistrationServiceTest.java

```diff
@@ -0,0 +1,177 @@
+package core.basesyntax;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationException;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceTest {
+    private static final String CORRECT_LOGIN = "1234567";
+    private static final String CORRECT_LOGIN_NEW = "7654321";
+    private static final String INCORRECT_LOGIN = "123";
+    private static final String BORDER_LOGIN = "123456";
+    private static final String EMPTY_LINE = "";
+    private static final String CORRECT_PASSWORD = "1234567";
+    private static final String INCORRECT_PASSWORD = "123";
+    private static final String BORDER_PASSWORD = "123456";
+    private static final int CORRECT_AGE = 22;
+    private static final int IN_CORRECT_AGE = 15;
```

Comment:
```suggestion
    private static final int INCORRECT_AGE = 15;
```
instead of having a separate constant, you can just use BORDER_AGE - 1